feat: notify admins on new releases

WheresAlice opened the pull request:
Notify admins that a new deployment was made to their server.  If the commit is in this GitHub repo then it will also include the latest git commit.

Example messages:

```
A new version of imok was deployed.
This commit was not found upstream
```

```
A new version of imok was deployed.
Commit: 18ede24c08f57e37ebb72fd1389232419fec56b9
Committer: WheresAlice
Message: feat: notify admins on new releases
```

I'm open to better messaging.

The only thing we know about the release is the git commit, and that git commit may or may not be in this repository.

kimadactyl commented:
Looks great, well done on getting this up!

I take it there is no meaningful way of testing this? I quite like the gherkins being a description of every feature the software has but understand this is prob a nightmare to write anything for.

I think the messages look fine. Questions about possible improvements:

- Is there a way to link to the last commit?
- Can it know if it's a release (i.e. 0.2.0?)
- Looks like we can use ** to bold stuff in telegram -- maybe just  bold the first line

WheresAlice commented:
I've added a link to the latest commit, though this might not mean much to the admins

I've also added tests to make sure the message being sent is sensible, but not that it actually gets sent at deploy time

In theory releases should be mentioned too, and the tests support that.  But we'd need to do another release to fully prove that.

Markdown in messages is a good idea, but seems to be a little broken in the bot API.  It causes messages to fail to be sent if they contain brackets.
